github-sync/github-data/users.yaml

```diff
@@ -25,8 +25,6 @@ users:
         role: member
       - name: architecture-doc-team
         role: member
-      - name: cosign-codeowners
-        role: member
```

Comment:
Not for everybody, this is @asraa but the diff is not smart enough.

Reply:
MAsra should remain as she is actively developing and reviewing PRs, especially around verification.

Reply:
I double-checked with her:
<img width="719" alt="Screenshot 2023-01-18 at 11 45 57 AM" src="https://user-images.githubusercontent.com/873857/213242473-06a921d0-bbb6-411f-9cf0-ae2b3f7f6030.png">